### ISSUE TITLE
Conflating governance of projects with governance of foundations

### ISSUE THREAD

**Issue by fungi** [sfosc/sfosc#47]
In the Governance and Behavior chapter, I worry that the distinction between projects and foundations is a bit blurry. For example, combining treatment of the OpenStack Project governance (elected technical committee and project team leads) with that of the OSF (board of directors) muddies the distinction in responsibilities and proximity to the community. This would have been akin to lumping Debian Project governance in with SPI Inc. operations, as the responsibilities of the OSF are primarily marketing functions, organizing events and handling trademarks (and decisions over the latter require agreement of the project technical leadership as well). At least where the OpenStack project is concerned the OSF board of directors has little if any say over the project's decisions, and so while it may appear on paper to have a lot of importance its controlling influence over the community is in fact quite limited.

**Comment by adamhjk** [comment]
Yes - it's complicated, which is why it winds up blurry. I would happily take something that makes it even more clear that the project direction is managed by the team leads. I think my criticism stands, though - the OSF board of directors has quite a bit of control, they just (wisely) don't really use it. That's why it appears on paper to have a lot of importance: it does.

**Comment by Beanow** [comment]
Thanks for your feedback. Sounds like there's no clear way to resolve this, so I feel like we need to close this for now. If you have concrete suggestions I'm happy to consider them :smile:

Feel free to re-open this issue or submit a PR.
Would you agree @nothingismagick?

**Comment by nothingismagick** [comment]
I agree @Beanow. But I still think that we shouldn't merely delegate this topic to the graveyard of closed issues, especially because it may arise again. Maybe it would make sense to add it somewhere else, such as an Addendum of Unresolved Discussions or something.

**Comment by fungi** [comment]
Thanks for the reminder, I meant to follow up sooner...

On the topic of perceived control of the OSF board of directors over the OpenStack project, the foundation bylaws "clearly" (for legal document definitions of clearly at least) state that the board cannot modify the bylaws which define its responsibilities and powers without agreement of the individual members (a strict majority assenting in a vote with at least 10% turnout of the currently ~20k folks who have signed up for a free individual membership and regularly vote in annual elections for individual directors). Also the board is not allowed to alter the bylaws appendix which delegates the OpenStack Technical Committee's operation and responsibilities without agreement of the sitting members of the TC.

So all that to say, I'm still unsure how to reflect such nuances in a generalized model, but I don't think that OpenStack is a great example of a project controlled by companies paying for seats on a foundation board. The OSF has (wisely) made provisions which put the individual members and democratically elected technical leaders in a position to defend the project against corporate overreach, and to encourage contributors in the project to ultimately decide how it will operate and be governed.

**Comment by ttx** [comment]
I agree... The "control" of the OSF board of directors has much more to do with employing most of the core contributors (influence by contribution). Its governance actually states that it has no control over technical matters. Yes, governance can change, but the bylaws go to great length to protect those sections and make them extra-hard to change. So overall, not the best example. Maybe the CNCF would be a better example, as their Technical Committee is completely selected by member companies ?

**Comment by Beanow** [comment]
Thank you for pitching in to clear this up :]
Bear with me as I neither wrote the original, nor am I very savvy on the specifics of OSF & OpenStack project, but I'll try to summarize my understanding.

First I just want to put out there, please don't consider what's written as a knock on real-world effectiveness or fairness. This is mainly an analysis using Rawls' theory, to find out if the _model_ has embedded Rawls' views on justice built into it. Which for SFOSC is useful to think about as it's a heuristic for broader application of the model and identifying potential issues. It's more of a first pass for understanding, not a fail/pass grading of how this deems the OpenStack implementation of this model to work actually.

So, most of this seems to revolve around the Platinum and Gold corporate memberships. For which two important concerns were mentioned in the text:
1. Platinum and Gold members combined could meet a 2/3 majority with purely corporate interests, potentially pushing out individual interests.
2. Those in the strongest starting position (wielding a big corp that can afford platinum, vs individuals) also gain the most benefits in this model.

To address those issues, you're saying. Additional protections for particularly sensitive points, like changing bylaws, are in place. And the types of influence that these seats afford you, are intentionally limited. Most importantly, technical governance of projects has been strongly isolated from these seats.

Of course, much simplified, but I hope I got the gist of it here.

**Comment by ttx** [comment]
Yes. But really I think Jeremy's point is that the Foundation Board of Directors is irrelevant in a discussion on open source project governance. In effect, the board of directors does not govern the open source project, technically or otherwise. That is what the OpenStack Technical Committee does (arguably a bad choice of words), and the Technical Committee is an elected representation of the contributors. The only power of the board is control of the trademark used by the project. It guess it could, in theory, decide to take that trademark away from "the contributors", forcing them to fork under a different name. But that has little to do with the open source project governance IMHO...

**Comment by Beanow** [comment]
Ah, there I would disagree. SFOSC tries to consider all benefits in a very broad sense (from patches, to money, to influence, to marketing exposure, to friendships, and so on). So even if influence on project governance is limited, aspects like holding trademarks / copyrights, budget allocation,

**Comment by ttx** [comment]
@Beanow makes sense. The platinum/gold members in the Board of Directors (and other direct or indirect sponsoring organizations) definitely have an influence over the project, even if the technical side of open source project governance is insulated from it by our governance model.

**Comment by Beanow** [comment]
Perhaps highlighting that and emphasizing the technical isolation, could clarify why the foundation seats are still brought into the analysis?
